sql/2013/01/webp-adoption-by-wordpress-version.sql

```diff
@@ -0,0 +1,134 @@
+# HTTP Archive query to get WebP adoption by WordPress (major) version.
```

Comment:
This file is in the wrong year-based folder (needs to be `2023`).

Reply:
oof, corrected.